tests/python/unittest/test_auto_scheduler_measure.py

```diff
@@ -67,6 +67,26 @@ def test_measure_local_builder_runner():
     assert mress[0].error_no == 0
 
 
+def test_measure_local_builder_rpc_runner():
+    dag, s0 = get_tiled_matmul()
+
+    if not tvm.runtime.enabled("llvm"):
+        return
```

Comment:
let's move these two lines to the beginning of this function

Reply:
Done.